yarn.lock

```diff
@@ -9,13 +9,14 @@
   dependencies:
     "@jridgewell/trace-mapping" "^0.3.0"
 
-"@architect/functions@^5.0.2":
```

Comment:
Please revert these changes, as you didn't update any `package.json`

Reply:
They somehow got back, so please remove the changes to `yarn.lock` again.

The easiest way to do this is to run
```sh
git checkout dev ./yarn.lock
```
in the root of the project.

Reply:
This is now the _opposite_ of your original complaint though. Originally, I updated it to 5.0.2 because that's what I got with a fresh install. You requested that was reverted and dealt with in a different issue which is what we went for.

If I revert this _again_, we'll be back to square one. There must be something going wrong here.

Reply:
I don't want it to be on a specific version, I just want to keep whatever we currently have in `dev` branch.
So probably this was changes in `dev` branch and that's why it looks like we would be back at square one 🤷‍♂️

But I just want to see **no** changes to `yarn.lock`

Reply:
I've plucked it out of the commit history. No diffs for `yarn.lock` now.